File: abp/src/Wallee.Boc.Vote.Application.Contracts/CandidateOrgUnits/CandidateOrgUnitUpdateDto.cs
using System;
using Volo.Abp.Domain.Entities;

namespace Wallee.Boc.Vote.CandidateOrgUnits
{
    public class CandidateOrgUnitUpdateDto : IHasConcurrencyStamp
    {
        /// <summary>
        /// 部门类别
        /// </summary>
        public CandidateOrgUnitCategory Category { get; set; }
        public Guid SuperiorId { get; set; }
        public Guid OrganizationUnitId { get; set; }
        public string ConcurrencyStamp { get; set; } = null!;
        public bool IsActive { get; set; }
    }
}

File: vote_vite_admin/src/openapi/models/CandidateOrgUnitUpdateDto.ts
/* istanbul ignore file */
/* tslint:disable */
/* eslint-disable */

import type { CandidateOrgUnitCategory } from './CandidateOrgUnitCategory';

export type CandidateOrgUnitUpdateDto = {
    category?: CandidateOrgUnitCategory;
    superiorId?: string;
    organizationUnitId?: string;
    concurrencyStamp?: string | null;
    isActive?: boolean;
};
